python/lsst/pipe/tasks/objectMasks.py

```diff
@@ -186,12 +192,13 @@ def read(fileName):
                     radius = NaN
                     width = NaN
                     height = NaN
-                    angle = NaN
+                    angle = 0.0*afwGeom.degrees
 
                     if _type == "box":
                         width = convertToAngle(param1, param1Unit, "width", fileName, lineNo)
                         height = convertToAngle(param2, param2Unit, "height", fileName, lineNo)
-                        angle = convertToAngle(param3, param3Unit, "angle", fileName, lineNo)
+                        if param3 is not None:
+                            angle = convertToAngle(param3, param3Unit, "angle", fileName, lineNo)
```

Comment:
You are basically doing work to optionally read in an angle if supplied but below there can only be an angle of zero, and it is only output as a warning. Should that be stronger than a warning as it is not going to do as the user requests? Or should we just take rotation out all the way?

Reply:
I went to change this to an error, but then noticed that it actually does end up throwing an error because of this. It's putting out warnings as it reads each line, and then if there were any problems it throws an error when it's done reading. I think failing late here is the right thing to do, as it allows the user to fix all the bad lines rather than one by one (nicely done, @RobertLuptonTheGood!).